plugins/woocommerce/includes/class-wc-regenerate-images.php

```diff
@@ -272,6 +272,12 @@ private static function get_full_size_image_dimensions( $attachment_id ) {
 			$imagedata['width']  = $imagedata['sizes']['full']['width'];
 		}
 
+		// The result of the earlier wp_get_attachment_metadata call is filterable, so we may not have height or
+		// width data at this point. 
```

Comment:
```suggestion
		// width data at this point.
```

Fixing some whitespace that escaped my IDE.